c10/core/impl/alloc_cpu.h

```diff
@@ -9,4 +9,14 @@ namespace c10 {
 C10_API void* alloc_cpu(size_t nbytes);
 C10_API void free_cpu(void* data);
 
+#ifdef USE_MIMALLOC_ON_MKL
+namespace mi_malloc_wrapper {
+C10_API void* c10_mi_malloc(size_t size);
+C10_API void* c10_mi_calloc(size_t count, size_t size);
+C10_API void* c10_mi_realloc(void* p, size_t newsize);
+C10_API void* c10_mi_malloc_aligned(size_t size, size_t alignment);
+C10_API void c10_mi_free(void* p);
+} // namespace mi_malloc_wrapper
+#endif
```

Comment:
Why do we need this?

Reply:
We have integrated mimalloc into `c10.dll` in pervious PR. In this PR we need optimize the kernels in `libtorch_cpu.dll`.
This C++ header is export mimalloc function from `c10.dll`, and let `libtorch_cpu.dll` import functions.

Reply:
What kernels in `libtorch_cpu.dll` need these exported API? I don't see the dependencies in this PR.

Reply:
OK, it looks good. I thought the allocations are all registered in c10.